src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
Don't forget to include access modifiers for your classes. In this case, it's good practice to declare the Ball class as public.